CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+- Upgrade Rails to 4.2.6, Ruby to 2.3.0, Rollbar to 2.8.3, Spring to 1.6.4
```

Comment:
please, remove lines below about upgrading rails and ruby
is it necessary to add info about upgrading gems like rollbar and spring to changelog?